### PR TITLE
BluetoothConnector 1.1.1 (new formula)

### PR DESCRIPTION
**Pull request by lapfelix**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds BluetoothConnector: https://github.com/lapfelix/BluetoothConnector

### CODE REVIEW

**Comment by lapfelix** [comment]
Thanks bot, I removed it

**Comment by lapfelix** [comment]
```Fetching git@github.com:lapfelix/SimpleCLI.git
error: failed to clone; Cloning into bare repository '/private/tmp/bluetoothconnector-20181128-69257-uq1adb/BluetoothConnector-1.1.1/.build/repositories/SimpleCLI.git-5655072109037723642'...
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.```

Why would this pass in Sierra but not in High Sierra and Mojave? 🤔

**Comment by lapfelix** [comment]
@BrewTestBot test this please

**Comment by lapfelix** [comment]
@BrewTestBot test this please

**Comment by lapfelix** [comment]
@BrewTestBot test this please

**Comment by lapfelix** [comment]
@BrewTestBot please test this

File: Formula/bluetoothconnector.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on :xcode => ["10.0", :build]
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "-Xswiftc", "-static-stdlib"
+    bin.install ".build/release/BluetoothConnector"
+  end
+
+  test do
+    system "#{bin}/BluetoothConnector"
```
[the remark this answers is not in the document]
Comment (lapfelix) [reply Formula/bluetoothconnector.rb]: Yes thank you that makes sense.I updated it and tested locally on my machine.

**Comment by fxcoudert** [comment]
Thanks @lapfelix

**Pull request closed without merging**